.github/workflows/release.yml

```
@@ -189,144 +154,43 @@ jobs:
-          name: ${{ env.REPO_NAME }}-windows-${{ matrix.package_arch }}
-          path: |
-            ${{ steps.stage.outputs.artifact_dir }}
-            ${{ steps.package.outputs.msi-path }}
+    uses: ./.github/workflows/build-and-package.yml
+    with:
+      platform: windows
+      runner: windows-latest
+      target: ${{ matrix.target }}
+      bin-name: ${{ needs.metadata.outputs.bin_name }}
+      version: ${{ needs.metadata.outputs.version }}
+      artifact-name: ${{ env.REPO_NAME }}-windows-${{ matrix.package_arch }}
```

actionlint (1.7.8)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)